### PR TITLE
added yaml files for running vllm on tpu

### PR DESCRIPTION
**Pull request by Edwinhr716**
## Description

This PR adds yaml files needed for vLLM on TPU GKE user guide

## Tasks

* [X] The [contributing guide](https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/main/.github/CONTRIBUTING.md) has been read and followed.
* [X] The samples added / modified have been fully tested.
* [X] Workflow files have been added / modified, if applicable.
* [X] Region tags have been properly added, if new samples.
* [X] All dependencies are set to up-to-date versions, as applicable.
* [X] Merge this pull-request for me once it is approved.

### CODE REVIEW

File: ai-ml/vllm-tpu/vllm-llama3-70b.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        gke-gcsfuse/memory-limit: "0"
+        gke-gcsfuse/ephemeral-storage-limit: "0"
+    spec:
+      serviceAccountName: KSA_NAME
+      nodeSelector:
+        cloud.google.com/gke-tpu-topology: 2x4
+        cloud.google.com/gke-tpu-accelerator: tpu-v6e-slice
+      containers:
+      - name: vllm-tpu
+        image: REGION_NAME-docker.pkg.dev/PROJECT_ID/vllm-tpu/vllm-tpu:latest
```
Comment (NimJay) [inline ai-ml/vllm-tpu/vllm-llama3-70b.yaml:42]: **Question:** In the tutorial, will the user be building this image? Curious where the source code exists for this image.

Comment (Edwinhr716) [reply ai-ml/vllm-tpu/vllm-llama3-70b.yaml:42]: Yes, the user will be building the image in the user guide

**Comment by Edwinhr716** [comment]
@NimJay does using upper case mean it functions as a variable in the userguide like it does in the yaml files in this one https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/hyperdisk-ml#create-from-hd?

**Review by NimJay — APPROVED** [review]
Thanks, @Edwinhr716, for creating these samples! 👏 

Looks good to me.

I trust that you've tested these samples (and the associated tutorial) appropriately.

**Comment by NimJay** [comment]
@Edwinhr716, good question.
Unfortunately, I believe those variables (like the ones in the image below) are only possible with in-line samples (not samples embedded by referencing files on GitHub):

Screenshot 2024-11-05 at 4 25 07 PM

I recommend just adding a comment in the sample itself similar to `# Replace with your value`.

**Pull request merged**